docs/conceptual/node/tx-pool.md

```diff
@@ -2,12 +2,67 @@
 title: Transaction Pool
 ---
 
-* What is the Transaction Pool?
 
-* What is priority?
+The transaction pool contains all transactions (signed & unsigned) broadcasted to the network. The pool is responsible for:
```

Comment:
Not sure how detailed you want to be in the first sentence, but the pool contains at most "all transaction broadcast to the network". And in almost all cases it contains strictly fewer transactions. Specifically it doesn't contain:
1. Transactions already included in blocks.
2. Transactions already determined to be invalid
3. Transactions broadcast, but not yet received by the local node

Reply:
Yeh I 'd like to keep the first sentence more sharp, but i've added in points (2) & (3) that you made! Happy for (1) to be woven in there if the majority would like it inserted.

Reply:
I think 1 is probably self-evident